mods/measure/init.lua

```diff
@@ -0,0 +1,50 @@
+local function deltas_as_string(pos1, pos2)
+	return("Deltas: X: "..math.abs(pos1.x-pos2.x).." | Y: "..math.abs(pos1.y-pos2.y).." | Z: "..math.abs(pos1.z-pos2.z))
+	end
+
+minetest.register_craftitem("measure:stick", {
+
+	description = "Measuring Stick",
+	inventory_image = "measure_stick.png",
+	stack_max = 1,
+
+	on_use = function(itemstack, user, pointed_thing)
+
+		if pointed_thing == nil then
+
+			minetest.chat_send_player(user:get_player_name(), "Punch a valid node!")
+			return -1
+			end
```

Comment:
`end` should always align with the statement that starts the block, which is `if` in this case. Please reduce the indentation of all occurrences of `end` by 1 tab-space.